docs/scanners.md

```diff
@@ -257,6 +257,9 @@ Scan rules:
 100031  DNS Email Spoofing [Script]
 100032  WordPress Username Enumeration [Script]
 100033  Server Side Template Injection [Script]
+100034  Information Disclosure - Google API Key [Script]
+100035  Information Disclosure - Java Stack Trace [Script]
```

Comment:
Is this considered information disclosure? Should this be called "Java Stack Trace Disclosure"?

Reply:
At some point we should normalise between Information Leak and Information Disclosure.